tests/delegation/e2e/steps/delegation.js

```diff
@@ -213,16 +219,20 @@ Given('I send {int} ADA from the {string} wallet to the {string} wallet', async
   await this.client.executeAsync((amount, senderName, receiverName, done) => {
     const walletSender = daedalus.stores.wallets.getWalletByName(senderName);
     const walletReceiver = daedalus.stores.wallets.getWalletByName(receiverName);
-    daedalus.stores.addresses
-      .getAddressesByWalletId(walletReceiver.id)
-      .then(addresses => {
-        daedalus.stores.wallets.sendMoneyRequest.execute({
-          address: addresses[0].id,
-          amount: amount * 1000000,
-          passphrase: 'Secret1234',
-          walletId: walletSender.id,
-        }).then(done)
-      })
+    if (walletSender && walletReceiver && walletSender.id && walletReceiver.id) {
+      daedalus.stores.addresses
+        .getAddressesByWalletId(walletReceiver.id)
+        .then(addresses => {
+          daedalus.stores.wallets.sendMoneyRequest.execute({
+            address: addresses[0].id,
+            amount: amount * 1000000,
+            passphrase: 'Secret1234',
+            walletId: walletSender.id,
+          }).then(done);
+        });
+    } else {
+      done();
```

Comment:
@yakovkaravelov this is not valid because we need to throw an error if some parameter is missing